### DISCUSSION TITLE
Cell Temperature in Carge Details

### DISCUSSION THREAD

**Discussion by baldohunter** [jheredianet/Teslamate-CustomGrafanaDashboards#61] [Ideas]
Sorry my knowledge is not enough to do it but could you add the average / min / max Cell temperature to the Charging details? That would be nice in Winter time.

something like this:

{
„matcher“: {
„id“: „byRegexp“,
„options“: „^cellavg_temp_.*“
},
„properties“: [
{
„id“: „displayName“,
„value“: „AVG Cell Temperature“
}
]
},

**Comment by jheredianet** [comment]
Hello, unfortunately the Tesla APIs do not give cell temperature values, therefore this data does not exist in the Teslamate database. 
The only temperatures we have available are the Outdoor, Inside and driver. 
As far as I know, the only way to know the real-time temperatures of the battery cells is using an ODB2 device.
